content/standardize-or-not-to-standardize.md

```diff
@@ -0,0 +1,86 @@
+Title: Common Engineering - To Standardize or Not to Standardize, view from a directors point
+Date: 2022-04-06
+Category: Posts
+Tags: journal, innovation
+Slug: standardize-or-not-to-standardize
+Author: Andre Kaminsiki
```

Comment:
Spelling mistake - should be 'Kaminski'

Reply:
@AndreKaminski ... My bad and appologies! Fixed.

I was actually staring at it yesterday and it felt wrong, but my brain did not compute.